components/swatchgroup/stories/swatchgroup.stories.js

```diff
@@ -1,16 +1,41 @@
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
+import { Sizes } from "@spectrum-css/preview/decorators";
 import { default as Swatch } from "@spectrum-css/swatch/stories/swatch.stories.js";
 import { version } from "../package.json";
-import { SwatchGroup } from "./swatchgroup.test.js";
+import { SwatchgroupGroup } from "./swatchgroup.test.js";
```

Comment:
Is it terrible that I changed this from swatchgroup, to "swatchgroup group?" 😆 It sounds silly but I think that matches our naming convention.